src/cai_lab_to_nwb/zaki_2024/utils/motion_correction.py

```diff
@@ -0,0 +1,56 @@
+from typing import List
+
+import numpy as np
+from neuroconv.tools import get_module
+from pynwb import NWBFile, TimeSeries
+from roiextractors.extraction_tools import DtypeType
+
+
+def add_motion_correction(
+    nwbfile: NWBFile,
+    motion_correction_series: np.ndarray,
+    one_photon_series_name: str,
+    convert_to_dtype: DtypeType = None,
```

Comment:
Why do we need this?

Reply:
So I can add the motion correction in the Converter once the OnePhotoSeries has been added

Reply:
My question was not very clear, I am talking about convert_to_dtype.

Reply:
Ah sorry! No we don't need it, I copied and paste it accidentally

Reply:
Can you reference the original implementation if you copied it from somewhere else?

Reply:
here: https://github.com/catalystneuro/pinto-lab-to-nwb/blob/9f3ed8be31c3077666bffaa57a3d718d1a82f535/src/pinto_lab_to_nwb/widefield/utils/motion_correction.py.

Reply:
Ah, is it our thing from one of our repos then. Thanks.